HMCL/src/main/resources/assets/lang/I18N.properties

```diff
@@ -124,12 +124,13 @@ account.methods.microsoft.snapshot.website=Official Website
 account.methods.offline=Offline
 account.methods.offline.name.special_characters=Recommended to use letters, numbers and underscores for naming
 account.methods.offline.name.invalid=Under normal circumstances, the game username can only contain English characters, numbers and underscores, and the length cannot exceed 16 characters.\n\
-\n\Some legitimate usernames: HuangYu, huang_Yu, Huang_Yu_123\n\
-\n\Some illegal usernames: Huang Yu, Huang-Yu_%%%, Huang_Yu_hello_world_hello_world\n\
-\n\If you believe that there is a corresponding mod or plugin on the server side to remove this restriction, you can ignore this warning.
+   Some legitimate usernames: HuangYu, huang_Yu, Huang_Yu_123\n\
+   Some illegal usernames: Huang Yu, Huang-Yu_%%%, Huang_Yu_hello_world_hello_world\n\
+   If you believe that there is a corresponding mod or plugin on the server side to remove this restriction, you can ignore this warning.
 account.methods.offline.uuid=UUID
 account.methods.offline.uuid.hint=UUID is the unique identifier for the game character in Minecraft. The way that it is generated might vary between different game launchers. Changing it to the one generated by other launchers allows you to keep your items in your offline account inventory.\n\
-\n\This option is for advanced users only. We do not recommend modifying this option unless you know what you are doing.
+\n\
+This option is for advanced users only. We do not recommend modifying this option unless you know what you are doing.
```

Comment:
Although the spaces before lines which is not the first one of this value is NOT necessary, adding two spaces will make this document more human-friendly. I suggest using the Code Format in your IDE.